k8s/namespaces/admin/aad-pod-identity/aad-pod-identity.yaml

```diff
@@ -0,0 +1,19 @@
+---
+apiVersion: helm.fluxcd.io/v1
+kind: HelmRelease
+metadata:
+  labels:
+    k8s-app: aad-pod-id
+  name: aad-pod-identity
+  namespace: admin
+spec:
+  releaseName: aad-pod-identity
+  chart:
+    repository: https://raw.githubusercontent.com/Azure/aad-pod-identity/master/charts
+    name: aad-pod-identity
+    version: 1.6.2
+  values:
+    installMICException: false
+    forceNameSpaced: true
+    nmi:
+      probePort: "8080"
```

Comment:
do we need this?

Reply:
Can change it (8085 default), included it as 8080 is what we have set currently

Reply:
we don't rely on that I would use the default